data/locale/en-US.ini

```diff
@@ -11,6 +11,12 @@ RestartCEF="Restart CEF"
 BrowserSource="Browser"
 CustomFrameRate="Use custom frame rate"
 RerouteAudio="Control audio via OBS"
+WebpageControlLevel="Webpage permissions"
+WebpageControlLevel.Level.None="Don't allow any access"
+WebpageControlLevel.Level.ReadOnly="Allow read-only access"
+WebpageControlLevel.Level.Basic="Allow basic control (Save replay buffer, etc.)"
+WebpageControlLevel.Level.Advanced="Allow advanced control (Switch scenes, Start/Stop replay buffer, etc.)"
+WebpageControlLevel.Level.All="Allow full control (Start/Stop streaming without warning, etc.)"
```

Comment:
I'd probably change this to "Page permissions" or "OBS API permissions" (for some reason "webpage" feels off to me in this context) with the additional context that this is access to OBS, eg:

None: No access to OBS
ReadOnly: Read-only access to OBS (current scene and status)
Basic: Basic access to OBS (save replay buffer)
Advanced: Advanced access to OBS (change scenes, start/stop replay buffer)
Full: Full access to OBS (start/stop streaming, recording, etc).

Otherwise I feel "access" is a bit vague as to what exactly it's providing access to, and "None" doesn't fit the style of the other options ("No access" would be better IMO). I feel it's important to get the wording right as this is a potentially very dangerous feature if users don't understand it properly and grant random websites full access.